### PR TITLE
[Doc] update the tutorial of xgboost4j-spark-gpu

### PR DESCRIPTION
**Pull request by wbo4958**

### CODE REVIEW

File: doc/jvm/xgboost4j_spark_gpu_tutorial.rst [hunk 1]
```diff
@@ -211,32 +215,31 @@ and the prediction for each instance.
 Submit the application
 **********************
 
-Here’s an example to submit an end-to-end XGBoost-4j-Spark-GPU Spark application to an
-Apache Spark Standalone cluster, assuming the application main class is Iris and the
-application jar is iris-1.0.0.jar
+Assuming that the application main class is "Iris" and the application jar is "iris-1.0.0.jar,"
```
Comment (trivialfis) [inline doc/jvm/xgboost4j_spark_gpu_tutorial.rst:218]:
```suggestion
Assuming that the application main class is "Iris" and the application jar is "iris-1.0.0.jar"`
```

File: doc/jvm/xgboost4j_spark_gpu_tutorial.rst [hunk 2]
```diff
@@ -211,32 +215,31 @@ and the prediction for each instance.
...
+provided below is an instance demonstrating how to submit the xgboost application to the Apache
```
Comment (trivialfis) [inline doc/jvm/xgboost4j_spark_gpu_tutorial.rst:219]:
```suggestion
provided below is an instance demonstrating how to submit the xgboost application to an Apache
```

File: doc/jvm/xgboost4j_spark_gpu_tutorial.rst [hunk 3]
```diff
@@ -211,32 +215,31 @@ and the prediction for each instance.
...
   main_class=Iris
   app_jar=iris-1.0.0.jar
 
   spark-submit \
     --master $master \
-    --packages ai.rapids:cudf:${cudf_version},com.nvidia:rapids-4-spark_2.12:${rapids_version},ml.dmlc:xgboost4j-gpu_2.12:${xgboost_version},ml.dmlc:xgboost4j-spark-gpu_2.12:${xgboost_version} \
+    --packages com.nvidia:rapids-4-spark_2.12:${rapids_version},ml.dmlc:xgboost4j-gpu_2.12:${xgboost_version},ml.dmlc:xgboost4j-spark-gpu_2.12:${xgboost_version} \
     --conf spark.executor.cores=12 \
-    --conf spark.task.cpus=1 \
+    --conf spark.task.cpus=12 \
```
Comment (trivialfis) [inline doc/jvm/xgboost4j_spark_gpu_tutorial.rst:233]: why the change?

Comment (wbo4958) [reply doc/jvm/xgboost4j_spark_gpu_tutorial.rst:233]: Yeah. Spark can't 100% ensure each training task will grab different GPU. I will add the stage-level scheduling in JVM package to fix that.

**Review by trivialfis — APPROVED** [review]
